lib/ReactViews/Custom/ChartCustomComponent.ts

```diff
@@ -338,7 +338,8 @@ export default abstract class ChartCustomComponent<
    * column contains a `<chart>`?
    * @param node The node to test
    */
-  private isFirstColumnOfChartRow(node: DomElement): boolean {
+  private isFirstColumnOfChartRow(_node: DomElement): boolean {
+    const node = _node as any;
```

Comment:
Casting to `any` as I think it is not worth the effort typing this properly. The nested typings is gnarly!